### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by dubinsky**
Made the tabs cleaner (spacing and color, etc), added a title and slogan in the proper place. Couldn't get the image to move from where it currently is. I can't figure out how to make <link> tag work since I can't seem to get the file path for the main.css file in our github. When I used <link src = "main.css"> it didn't work. If you know how to get the correct file path/name, that would be great :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 </head>
 <script>
 // JS for the tabs
-function openCity(evt, cityName) {
+function openCity(evt, tab) {
```
Comment (dubinsky) [inline index.html]: please rename this away from `...City`

File: index.html [hunk 2]
```diff
@@ -145,22 +122,25 @@ <h1 class = title>MATH WORLDS</h1>
...
 	</div>
 	</p>
-	
-	<img class = logo, src = "file:///C:/Users/The%20Critic/Downloads/Logo-Final-outlines.svg", width = 250px, height = 300vw, alt = "Happy Robot">
 <-->
 <div class="tab">
-  <button class="tablinks" onclick="openCity(event, 'Home')">Home</button>
-  <button class="tablinks" onclick="openCity(event, 'AboutUs')">About Us</button>
-  <button class="tablinks" onclick="openCity(event, 'MWGames')">MW Games</button>
+  <button class="tablinks" onclick="openCity(event, 'Home')"><strong>Home</strong></button>
```
Comment (dubinsky) [inline index.html]: please do not use presentation tags like `<strong>`; style button labels using CSS.

File: index.html [hunk 3]
```diff
@@ -191,7 +171,14 @@ <h3>MW Games</h3>
 	</ul>
   
   </p>
+</div> 
+<div id="ContactUs" class = "tabcontent">
+	<h3>Contact Us</h3>
+	<p> Contact us at insert contact info here
+	</p>
 </div>
+	<img class = logo, src = "file:///C:/Users/The%20Critic/Downloads/Logo-Final-outlines.svg", width = 150px, height = 150vw, alt = "Happy Robot">
```
Comment (dubinsky) [inline index.html]: do not use absolute paths or paths that are only valid on your machine;
this needs to work when it is deployed in the cloud.
Use paths that are relative to the HTML file or absolute path from the root of the deployment.

**Comment by dubinsky** [comment]
> Couldn't get the image to move from where it currently is.
Not sure what you mean by that.

> I can't figure out how to make tag work since I can't seem to get the file path for the main.css file in our github.
> When I used it didn't work. If you know how to get the correct file path/name, that would be great :)
`main.css` has absolute path `/main.css` and relative to the `index.html` its path is `./main.css` or `main.css`.
When developing locally, do you use some kind of a web server to serve the pages you are working with?
If not, you need to start doing this, or paths above won't work on your machine...

**Comment by dubinsky** [comment]
@CriticalWasTaken you forgot to create a pull request from this branch.
I did it for you :)

File: index.html (hunk 3 above)
Comment (CriticalWasTaken) [reply index.html]: Sorry, but I'm not sure how to locate such an absolute path to do so.

File: index.html (hunk 2 above)
Comment (CriticalWasTaken) [reply index.html]: Will fix in the next pull request im'h.

File: index.html (hunk 1 above)
Comment (CriticalWasTaken) [reply index.html]: Sorry, missed this. Will fix in next pull request im'h,

File: index.html (hunk 3 above)
Comment (dubinsky) [reply index.html]: > Sorry, but I'm not sure how to locate such an absolute path to do so.

As I wrote above:

`main.css` has absolute path `/main.css` and relative to the `index.html` its path is `./main.css` or `main.css`.
When developing locally, do you use some kind of a web server to serve the pages you are working with?
If not, you need to start doing this, or paths above won't work on your machine...

File: index.html (hunk 1 above)
Comment (dubinsky) [reply index.html]: > Sorry, missed this. Will fix in next pull request im'h,

Please do it in THIS pull request, not the next one.

File: index.html (hunk 3 above)
Comment (CriticalWasTaken) [reply index.html]: When developing locally, I haven't been using <link> to access the css from somewhere else. Maybe we can discuss this at some point, to make sure I don't mess this all up?

Comment (dubinsky) [reply index.html]: To not mess this all up, you need to use a local web server while working on it. So, again: are you?

For explanation why it is needed and some suggestions how to go about it, see https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server

Comment (CriticalWasTaken) [reply index.html]: I don't fully understand what this local testing server does or is supposed to do. Can we discuss this over Shabbos?

Comment (CriticalWasTaken) [reply index.html]: So no, I don't think I'm currently using this.

Comment (dubinsky) [reply index.html]: > I don't fully understand what this local testing server does or is supposed to do. Can we discuss this over Shabbos?

If you are allowed to discuss such topics on Shabbos - sure :)

File: index.html (hunk 1 above)
Comment (CriticalWasTaken) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (CriticalWasTaken) [reply index.html]: Done. Why is this? Purely convention?

File: index.html (hunk 1 above)
Comment (dubinsky) [reply index.html]: Please rename `openCity` to something that does not mention cities.

File: index.html (hunk 2 above)
Comment (dubinsky) [reply index.html]: Hmm... "done" in what sence? Please DO NOT USE PRESENTATION TAGS like `strong`; style with CSS.

> Why is this?

To separate structure (HTML) from presentation (CSS).
This is also why CSS needs to be in a separate file.

File: index.html (hunk 1 above)
Comment (CriticalWasTaken) [reply index.html]: Oh no! Am I editing the wrong branch? I fixed this issue, but not here??

File: index.html (hunk 2 above)
Comment (CriticalWasTaken) [reply index.html]: I did this, but was it in the wrong branch?

File: index.html [hunk 4]
```diff
@@ -145,22 +123,25 @@ <h1 class = title>MATH WORLDS</h1>
...
 	</p>
-	
-	<img class = logo, src = "file:///C:/Users/The%20Critic/Downloads/Logo-Final-outlines.svg", width = 250px, height = 300vw, alt = "Happy Robot">
 <-->
 <div class="tab">
-  <button class="tablinks" onclick="openCity(event, 'Home')">Home</button>
-  <button class="tablinks" onclick="openCity(event, 'AboutUs')">About Us</button>
-  <button class="tablinks" onclick="openCity(event, 'MWGames')">MW Games</button>
+  <button class="tablinks" onclick="openTab(event, 'Home')">Home</button>
+  <button class="tablinks" onclick="openTab(event, 'AboutUs')"About Us</button>
```
Comment (dubinsky) [inline index.html]: Some closing angle brackets ">" seem to be missing...

File: index.html (hunk 3 above)
Comment (dubinsky) [reply index.html]: remove your-local-machine-specific paths

**Pull request merged**